### DISCUSSION TITLE
Requests for Topic and Feedback on OSCAL Model Engineering Meeting 20230223

### DISCUSSION THREAD

**Discussion by aj-stein-nist** [usnistgov/OSCAL#1661] [General]
Hello,

Greetings from the NIST OSCAL Team. The team will host our bi-weekly OSCAL Model Engineering Meeting on, February 23, 2023, at 10-11 AM ET.

Details about how to join the meeting are on the OSCAL website on this page.

https://pages.nist.gov/OSCAL/contribute/model-engineering-meeting/

The tentative agenda will include the topics below.

- Review Meeting Ground Rules
- OSCAL Status Summary
- Review current active issues
- Review new issues
- Revisit [usnistgov/OSCAL#1548: Improve Document UUID and Object-Level UUID Guidance](https://github.com/usnistgov/OSCAL/issues/1548)
- Recap on follow-up work on types of control origination
- Open discussion

Most importantly, the team wants your feedback, fellow community member! There is a subset of GitHub issues the team maintains to highlight those issues where we most want your feedback. Please comment on them before, during, or after the meeting.

https://github.com/orgs/usnistgov/projects/25/views/11

Do you not see an existing issue or a new topic that is not in that list? If you want to ask in advance to discuss it, feel free to reply to this email or the following thread on the GitHub Discussions board.

**Comment by aj-stein-nist** [comment]
@vmangat you had expressed interest on discussing your perspectives on implementing with UUID management with your software and current stable release of OSCAL, per comments in #1548. Are you interested in doing so next Thursday?

**Comment by vmangat** [comment]
Yes please sounds good I have carved out the time for that.
